PROTOCOL.md

```diff
@@ -2334,17 +2364,45 @@ content-type: application/x-ndjson; charset=utf-8
 ### Timestamp Format
 Accepted timestamp format by a delta sharing server: in the ISO8601 format, in the UTC timezone, such as `2022-01-01T00:00:00Z`.   
 
-## API Response Format
+## Delta Sharing Capabilities Header
+This section explains the details of delta sharing capabilities header, which is introduced to help 
+delta sharing to catch up with features in [delta protocol](https://github.com/delta-io/delta/blob/master/PROTOCOL.md).
+
+The key of the header is **delta-sharing-capabilities**, the value is semicolon separated capabilities. 
+Each capability is in the format of "key=value1,value2", values are separated by comma.
+Example: "responseformat=delta;readerfeatures=deletionvectors,columnmapping". All key and values should
+be case-insensitive when processed by server.
 
-This section discusses the API Response Format returned by the server.
+This header can be used in the request for [Query Table Metadata](#query-table-metadata), 
+[Query Table](#read-data-from-a-table), and [Query Table Changes](#read-change-data-feed-from-a-table).
+
+Currently, two capabilities are introduced. 
+
+### responseformat
```

Comment:
responseFormat ? It's case insensitive, but makes it easier to understand at a glance that it is 2 words. Similarly for readerFeatures

Reply:
updated.